flang/test/Lower/intrinsic-procedures/TRIM.f90

```diff
@@ -0,0 +1,39 @@
+! RUN: bbc -emit-fir %s -o - | FileCheck %s
+
+! CHECK-LABEL: @_QPlen_trim_test
```

Comment:
This test includes both trim and len_trim.

Reply:
My mistake, fixing!